### PR TITLE
Remove `.ui` files from being the base class

### PR DESCRIPTION
**Pull request by sithlord48**
Quick sweep to remove the ui files from being the base class of the  object class

### CODE REVIEW

File: src/gui/src/ActionDialog.h [hunk 1]
```diff
@@ -20,36 +20,38 @@
...
-    return m_pKeySequenceWidgetHotkey;
-  }
+  const KeySequenceWidget *sequenceWidget() const;
   const ServerConfig &serverConfig() const
   {
     return m_ServerConfig;
   }
 
 private:
+  std::unique_ptr<Ui::ActionDialog> ui_;
```
Comment (nbolton) [inline src/gui/src/ActionDialog.h:54]:
```suggestion
  std::unique_ptr<Ui::ActionDialog> m_ui;
```

Comment (nbolton) [inline src/gui/src/ActionDialog.h:54]:
```suggestion
  std::unique_ptr<Ui::ActionDialog> m_ui;
```

Please follow the code style and use `m_` style members: https://github.com/deskflow/deskflow/wiki/Code-Style#naming

File: src/gui/src/AddClientDialog.h [hunk 2]
```diff
@@ -17,13 +17,15 @@
...
 #pragma once
 
-#include "ui_AddClientDialogBase.h"
-
 #include <QDialog>
 
 class QPushButton;
 class QLabel;
 
+namespace Ui {
```
Comment (nbolton) [inline src/gui/src/AddClientDialog.h:25]: Can we use the code style for namespaces here or is this a fixed Qt name? https://github.com/deskflow/deskflow/wiki/Code-Style#naming

**Review by nbolton — CHANGES_REQUESTED** [review]
Good PR. Only a couple of code style comments.

File: src/gui/src/ActionDialog.h (hunk 1 above)
Comment (sithlord48) [reply src/gui/src/ActionDialog.h:54]: id rather not . we have limited space and extra chars are gonna eat my limited line space.

I plan to update the guide line for this naming convention any way 

1. All the object on your form are pointers so adding p is just stupid
2. ALL Qt members with a ui files use the name space `ui` if i am changing `ui_` its to `ui`
3. Limited line space long ass name prefixes =  more poorly linted lines.

File: src/gui/src/AddClientDialog.h (hunk 2 above)
Comment (sithlord48) [reply src/gui/src/AddClientDialog.h:25]: We will not be doing that as that is not Qt like

File: src/gui/src/ActionDialog.h (hunk 1 above)
Comment (sithlord48) [reply src/gui/src/ActionDialog.h:54]: honestly rather change this to just `ui` since that is the Qt standard for the ui dialog class name.

**Review by nbolton — APPROVED** [review]

**Pull request merged**